partner_company_group/models/account_invoice.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    company_group_id = fields.Many2one(
+        related='partner_id.company_group_id',
```

Comment:
readonly?

Reply:
@mpanarin No need to `readonly = True` when a field is related.

Everywhere

Reply:
You will be able to change the field and, therefore, changing the value on a different record completely. This is not very user-friendly and obscure, as well as should be avoided. 
So I still think they should be readonly

Reply:
@mpanarin related fields are readonly=true as a default in v12. although i would have forgotten it.
https://github.com/odoo/odoo/blob/12.0/odoo/fields.py#L427

Reply:
@mpanarin In Odoo Base version 12, related always `readonly = True` look like a `compute/ Functional` fields, so no need to added this type of attribute